scripts/o3de/o3de/utils.py

```diff
@@ -781,3 +782,63 @@ def on_terminate(proc) -> None:
     except Exception:  # purposefully broad
         process_logger.error("Unexpected exception while waiting for processes to terminate, with stacktrace:", exc_info=True)
 
+def load_template_file(template_file_path, template_env, read_encoding:str = 'UTF-8', encoding_error_action:str='ignore') -> str:
```

Comment:
The typing information for the first two parameters are missing.
Should the templat_file_path parameter be a `pathlib.Path` and templat_env be a `dict`
```suggestion
def load_template_file(template_file_path : pathlib.Path, template_env :dict, read_encoding:str = 'UTF-8', encoding_error_action:str='ignore') -> str:
```